src/chrome/content/rules/Yandex.com.xml

```diff
@@ -297,63 +114,12 @@ Fetch error: http://www.weather.yandex.com/ => https://www.weather.yandex.com/:
 	<target host="video.yandex.com" />
 	<target host="vremya.yandex.com" />
 	<target host="watch.yandex.com" />
-	<target host="weather.yandex.com" />
-	<target host="www.weather.yandex.com" />
 	<target host="webmaster.yandex.com" />
 	<target host="widgets.yandex.com" />
 	<target host="wordstat.yandex.com" />
-	<target host="www.yandex.com" />
 	<target host="xml.yandex.com" />
 
-	<!--	Special cases:
-				-->
-	<target host="www.browser.yandex.com" />
-	<target host="www.constructor.maps.yandex.com" />
-	<target host="site.yandex.com" />
-	<target host="www.site.yandex.com" />
-	<target host="www.video.yandex.com" />
-	<target host="www.webmaster.yandex.com" />
```

Comment:
I think I did it because it's unlikely anyone would type www.site.yandex.ru or something similar, and there are likely no links to `www` versions of these domains.